protocol-specs/generators/block_processing_second_signature/index.js

```diff
@@ -292,12 +303,16 @@ const generateTestCasesInvalidBlockSecondSignatureTxSecondTime = () => {
 
 	const secondSignature =
 		'erupt sponsor rude supreme vacant delay salute allow laundry swamp curve brain';
+	const secondPassphraseObject = registerSecondPassphrase({
```

Comment:
do we need to use this function at all?

Reply:
Removed and replaced by object and `getKeys`